prep-javascript-functions/main.js

```diff
@@ -0,0 +1,55 @@
+function addTwoNumbers(x, y) {
+  return x + y;
+}
+
+var sum = addTwoNumbers(4, 4);
+console.log('sum', sum);
+
+function convertHoursToMinutes(x) {
+  return x * 60;
+}
+
+var multiply = convertHoursToMinutes(3);
+console.log('multiply', multiply);
+
+function getGreeting(name) {
+  return 'Hello' + ' ' + name + '!';
+}
+
+var greeting = getGreeting('Sunshine');
+console.log('greeting', greeting);
+
+function addAndMultiplyBy5(a, b) {
+  return (a + b) * 5;
+}
+
+var addMultiply = addAndMultiplyBy5(3, 5);
+console.log('addMultiply', addMultiply);
+
+function subtractTwoNumbers(c, d) {
+  return c - d;
+}
+
+var subtract = subtractTwoNumbers(20, 5);
+console.log('subtract', subtract);
+
+function getCircleCircumference(r) {
+  return 2 * (22 / 7) * r;
```

Comment:
The JavaScript `Math` object has some convenient constants built into it.

```js
return 2 * Math.PI * r;
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math

Reply:
Good to know! I applied the changes :)